### ISSUE TITLE
Feature request: Max Fan speed

### ISSUE THREAD

**Issue by Zyarra** [Rem0o/FanControl.Releases#1454]
Hey!
Since some of my fans(Due the grid on the pc case etc) are louder than anothers, I would like to set max Rpm%, so it reacts to my cpu temperature, yet never reaches above lets say 50%.
I think I'm not the only one with different acoustics but "same" curve requirement.
Right now I have set these to AVG multiple sensors offset -25, but since I allow my aio/gpu fans to go higher than 50%, this fan can still reach occasionally unwanted RPM.

TLDR:
Same as the minimum% just maximum%.

**Comment by bromix** [comment]
Looking for the same feature/option. My 5x140 Fans should run at max. 1200 rpms, but sometime they report running at 2400 rpms. It would help me to configure the fan curve, but define a cutoff max. rpms. Now I have to test via the percentage to hit almost the 1200 rpms.

**Comment by Zyarra** [comment]
> Looking for the same feature/option. My 5x140 Fans should run at max. 1200 rpms, but sometime they report running at 2400 rpms. It would help me to configure the fan curve, but define a cutoff max. rpms. Now I have to test via the percentage to hit almost the 1200 rpms.

I noticed its already available but only in the linear curve. But that also only in percentage, not in RPM, i don't think you can set RPM, its pwm, it works in % of maximum,no?

**Comment by bromix** [comment]
You're right with rpm and pwm. I used the tool for identifying the fans and while playing with the percentages I watched the reported rpm of each fan. To run my fans at a optimal speed (rpm) I must set the pwm to ~35%, above it's like a yet turbine ^^ While playing games and monitoring the temperture with MSI Afterburner it seems to work (cooling). I got the fans quiet and cooling was the same as bevor.

**Comment by Zyarra** [comment]
> You're right with rpm and pwm. I used the tool for identifying the fans and while playing with the percentages I watched the reported rpm of each fan. To run my fans at a optimal speed (rpm) I must set the pwm to ~35%, above it's like a jet turbine ^^ While playing games and monitoring the temperture with MSI Afterburner it seems to work (cooling). I got the fans quiet and cooling was the same as bevor.

Try linear fan curve. Not exactly what you(and me) want, but it works till Rem0o gives us the option :) At least for case fans its good enough.
